docs/core/windows-prerequisites.md

```diff
@@ -13,45 +13,117 @@ ms.assetid: c33b1241-ab66-4583-9eba-52cf51146f5a
 > Make sure that your Windows installation is up-to-date and includes hotfix [KB2533623](https://support.microsoft.com/help/2533623) installed through Windows Update.
 
-## Prerequisites with Visual Studio 2017
+[.NET Core 2.x](#tab/netcore2x)
 
-You can use any editor of your choice to develop .NET Core applications using the .NET Core SDK. However, if you want to develop .NET Core applications on Windows in an integrated development environment, you can use [Visual Studio 2017](#visual-studio-2017).
+**.NET Core 2.0 Supported Windows Versions**
 
-> [!IMPORTANT]
-> Even though, it's possible to use Visual Studio 2015 with a preview version of the .NET Core tooling, these projects will be *project.json*-based, which is now deprecated. Visual Studio 2017 uses project files based on MSBuild. For more information about the format changes, see [High-level overview of changes](./tools/cli-msbuild-architecture.md).
+.NET Core 2.0 is supported on the following versions of Windows:
 
-To use Visual Studio 2017 to develop .NET Core apps, you'll need to have the latest version of Visual Studio installed with the **.NET Core cross-platform development** toolset (in the **Other Toolsets** section) selected.
-![Screenshot of Visual Studio 2017 installation with the ".NET Core cross-platform development" workload selected](./media/windows-prerequisites/vs_workloads.jpg)
+* Windows 7 SP1
+* Windows 8.1
+* Windows 10
+* Windows Server 2008 R2 SP1 (Full Server or Server Core)
+* Windows Server 2012 SP1 (Full Server or Server Core)
+* Windows Server 2012 R2 (Full Server or Server Core)
+* Windows Server 2016 (Full Server, Server Core, or Nano Server)
+
+See the [.NET Core Release Notes](https://github.com/dotnet/core/blob/master/release-notes/2,0/2.0.md) for the full set of supported operating systems.
+
+**.NET Core 2.0 Dependencies**
+
+.NET Core 2.0 requires the Visual C++ Redistributable when running on Windows versions earlier than Windows 10 and Windows Server 2016. This dependency is automatically installed for you if you use the .NET Core 2.0 installer.
+
+However, if you are installing .NET Core 2.0 via the [installer script](./tools/dotnet-install-script.md), or deploying a self-contained .NET Core 2.0 application; you need to manually install the [Microsoft Visual C++ 2015 Redistributable Update 3](https://www.microsoft.com/en-us/download/details.aspx?id=52685).
+
+> [!NOTE]
+> <em>For Windows 7 and Windows Server 2008 machines only:</em><br>
+> Make sure that your Windows installation is up-to-date and includes hotfix [KB2533623](https://support.microsoft.com/help/2533623) installed through Windows Update.
+
+## Developing .NET Core Applications in Visual Studio 2017
+
+You can use any editor of your choice to develop .NET Core applications using the .NET Core SDK. However, if you want to develop .NET Core applications on Windows in an integrated development environment, you can use [Visual Studio 2017](#visual-studio-2017).
```

Comment:
I think we need to keep the SDK part in the sentence otherwise you can't develop using other editors, no?